prebuilt-tasks/pom.xml

```diff
@@ -134,5 +134,22 @@
             <artifactId>spring-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <!-- Start libraries for Azure access -->
+        <dependency>
+            <groupId>com.azure</groupId>
+            <artifactId>azure-core</artifactId>
+            <version>1.37.0</version>
```

Comment:
please use properties for versions like we do it for other dependencies.

Reply:
Done